@xen-orchestra/lite/src/components/form/FormInputWrapper.vue

```diff
@@ -59,7 +59,10 @@ const props = withDefaults(
   { disabled: undefined }
 )
 
-const slots = useSlots()
+const slots = useSlots() as {
+  label?(): any
+  default(): any
+}
```

Comment:
It would be better to use `defineSlots`

```suggestion
const slots = defineSlots<{
  default(): any
  label?(): any
}>()
```

But when looking at the code, we can notice that there is no longer `label` slot in the template.

So it's probably preferable to remove this part and update `provide` (line 79) to use `props.label` instead of `slots.label`

Reply:
Yes I would like to use defineSlots but i was confused because I read that useSlots used to dynamically access slots passed to the component. So I didn't want to take the risk of breaking something

Reply:
Yes, `useSlots` will probably be rarely (if at all) used since `defineSlots` (Vue 3.3+) returns the same object.